ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileClustersTable.tsx

```diff
@@ -145,50 +165,66 @@ function ProfileClustersTable({
                                                     text="check"
                                                     status="manual"
                                                     count={manualCount}
+                                                    disabled={shouldDisableIcon(['Manual'])}
                                                 />
                                             </Td>
                                             <Td dataLabel="Other status" modifier="fitContent">
                                                 <StatusCountIcon
                                                     text="check"
                                                     status="other"
                                                     count={otherCount}
+                                                    disabled={shouldDisableIcon([
+                                                        'Error',
+                                                        'Info',
+                                                        'Not Applicable',
+                                                        'Inconsistent',
+                                                        'Unset Check Status',
+                                                    ])}
                                                 />
                                             </Td>
                                             <Td dataLabel="Compliance">
-                                                <Progress
-                                                    id={progressBarId}
-                                                    value={passPercentage}
-                                                    measureLocation={
-                                                        ProgressMeasureLocation.outside
-                                                    }
-                                                    aria-label={`${clusterName} compliance percentage`}
-                                                />
-                                                <Tooltip
-                                                    content={
-                                                        <div>
-                                                            {`${passCount} / ${totalCount} checks are passing for this cluster`}
-                                                        </div>
-                                                    }
-                                                    triggerRef={() =>
-                                                        document.getElementById(
-                                                            progressBarId
-                                                        ) as HTMLButtonElement
-                                                    }
-                                                />
+                                                {isComplianceStatusFiltered() ? (
```

Comment:
What are pro and con for future refactoring as a shared element, as you did for `StatusCountIcon`?

Reply:
I think it's a great idea. Went ahead and created a `ComplianceProgressBar` component.